examples/squeezenet/utils.js

```diff
@@ -56,6 +62,8 @@ class Utils {
     let start = performance.now();
     let result = await this.model.compute(this.inputTensor, this.outputTensor);
     let elapsed = performance.now() - start;
+    if (this.postOptions.softmax)
+      this.outputTensor = this.softmax(this.outputTensor);
```

Comment:
are we able to insert a softmax layer instead of computing it in JS?

Reply:
Absolutely. Now I implemented softmax as an extra layer. (and I will upload it later today)

I think it would be better to add an constructor option that allows user to decide if it should automatically append a softmax layer.
```javascript
constructor(onnxModel, backend, options = {}) { ... }
```

There are two benefits to doing so:
- More general to other applications other than image classification.  
- Easier to test. Though it's possible to add a softmax to the raw outputs of the test data set. But in some cases, such a layer will make lots of outputs to be zero so that we cannot compare two results effectively.